03-Raster-Calculations-In-R.Rmd

```diff
@@ -96,24 +109,15 @@ buildings, etc with the influence of ground elevation removed.
 <figure>
     <a href="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png">
     <img src="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png"></a>
-    <figcaption>A canopy height model represents the difference between the digital 
-    surface model and a digital terrain model. It represents the actual height 
-    of the trees with the influence of elevation, removed.</figcaption>
+    <figcaption> A canopy height model represents the difference between the
+    digital surface model and a digital terrain model. It represents the actual
+    height of the trees with the influence of elevation removed. Image source:
+    NEON, Inc.</figcaption>
```

Comment:
Even if image copy rights are going to change next April :( But that is going to be some fun re-branding anyway ;)